### PR TITLE
[#132894] ical link download fixes

### PR DESCRIPTION
**Pull request by meara**
- remove underscore from appearing around + icon
- allow facility staff "acting as" another user to download ical

### CODE REVIEW

**Comment by jhanggi** [comment]
When you merge, can you give the commit a little more descriptive title?

File: app/controllers/reservations_controller.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class ReservationsController < ApplicationController
 
   customer_tab  :all
   before_action :authenticate_user!, except: [:index]
-  before_action :check_acting_as, only: [:switch_instrument, :show, :list]
+  before_action :check_acting_as, only: [:switch_instrument, :list]
```
Comment (waggles) [inline app/controllers/reservations_controller.rb]: is this related?

Comment (meara) [inline app/controllers/reservations_controller.rb]: Yes. The `:check_acting_as` was blocking admins ordering on behalf of users from hitting the show action, which is what the calendar icon links to for its download.

**Comment by meara** [comment]
@jhanggi added a fix for those already initialized warnings in here
